OpenEdXMobile/src/main/java/org/edx/mobile/http/HttpResponseStatusException.java

```diff
@@ -1,13 +1,37 @@
 package org.edx.mobile.http;
 
+import android.support.annotation.NonNull;
+
+import retrofit2.Response;
+
+/**
+ * Signals an HTTP status error.
+ */
 public class HttpResponseStatusException extends Exception {
```

Comment:
I don't see how "Status" is "limiting its scope", but I have shortened it to `HttpStatusException` instead in commit f2d544e247e9c4c9d975adc81e01f475f1c48070.